examples/gke-security-best-practices/bastion.tf

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+/*
+  This file manages ssh bastion host resources for the cluster
+*/
+
+//Create a bastion host VM to access our GKE cluster
```

Comment:
I'd really like to see consistency with comments. You have #, /../ and // in the same file, seemingly at random. You also have some comments with a space after // and some without. Some are full sentences, some are not.

Reply:
Thank you dparrish I will address this.

Reply:
switched to pound sign '#'